src/store/migrations/update-default-name.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  migrate(state, store) {
+    return new Promise(async resolve => {
+      const {
+        commit,
+        state: { network },
+      } = store;
+      const networks = {
+        ...network,
+        ...state.userNetworks.reduce((p, n) => ({ ...p, [n.name]: { ...n } }), {}),
+      };
+
+      const { networkId } = networks[state.current.network];
+      setTimeout(() => {
+        state.subaccounts.forEach(({ aename, publicKey }) => {
+          if (aename) commit('names/setDefault', { address: publicKey, name: aename, networkId });
```

Comment:
Can't this be done manually? To make this migration sync and to don't depend on vuex commit that can be changed later.